input/images-source/FHIR-resource-careplan-rheumaticfever-pilot.puml

```diff
@@ -0,0 +1,317 @@
+@startuml FHIR-resource-careplan-rheumaticfever-pilot
+!include plantuml-include/std-skinparams-classdiagram.iuml
+
+top to bottom direction
+
+skinparam dpi 400
+
+scale 500 width
```

Comment:
Not sure if this is causing scale issues on the rendered image 
<img width="1413" alt="image" src="https://github.com/tewhatuora/cinc-fhir-ig/assets/18519309/5c0786d4-0a16-4608-b818-8c5aecd89336">

Reply:
Not sure why the third image is rendering like this - I'm going to have a look at this separately later today